tests/integration.rs

```diff
@@ -266,3 +266,30 @@ fn test_ctx_flags() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn test_server_event() -> Result<()> {
+    let port: u16 = 6490;
+    let _guards = vec![start_redis_server_with_module("server_events", port)
+        .with_context(|| "failed to start redis server")?];
+    let mut con =
+        get_redis_connection(port).with_context(|| "failed to connect to redis server")?;
+
+    redis::cmd("flushall")
+        .query(&mut con)
+        .with_context(|| "failed to run string.set")?;
```

Comment:
The same copy-paste typo is also found in other locations
```suggestion
        .with_context(|| "failed to run flushall")?;
```